packages/spark/sodasql/dialects/spark_dialect.py

```diff
@@ -230,11 +232,69 @@ def sql_test_connection(self) -> bool:
             logger.warning(f'{self.database} does not contain any tables.')
         return True
 
-    def sql_columns_metadata(self, table_name: str):
+    def show_columns(self, table_name: str) -> List[str]:
+        """
+        Show columns in table.
+
+        Parameters
+        ----------
+        table_name : str
+            The table to show columns in.
+
+        Returns
+        -------
+        out : List[str]
+            The column names.
+        """
+        qualified_table_name = self.qualify_table_name(table_name)
```

Comment:
Something for a refactor once: let all methods on the dialect expect the qualified table name instead of creating this table name within the methods.